src/index.ts

```diff
@@ -0,0 +1,235 @@
+/**
+ * Type for (x, y) coordinate pair.
+ */
+ export type Point = { x: number, y: number };
```

Comment:
Codacy has a fix for the issue: [Replace `·export·type·Point·=·{·x:·number,` with `export·type·Point·=·{·x:·number;`](https://app.codacy.com/gh/icooper/curvereduce-js/pullRequest?prid=9960097)
```suggestion
export type Point = { x: number; y: number };
```